### ISSUE TITLE
Reset fullscreen state of windows after exiting overview

### ISSUE THREAD

**Issue by KZDKM** [KZDKM/Hyprspace#20]
Just putting here in case I forget later. The plugin should restore the fullscreen state of all windows after exiting overview so workflows with multiple fullscreen workspaces would not be messed up by opening overview.

**Comment by alba4k** [comment]
@KZDKM you do know is quite broken when stuff like firefox is fullscreen, right? window is restored as fullscreen but the internal fullscreen state isn't triggered (in firefox's case, all bars remain visible, even when say a video was playing)

**Comment by KZDKM** [comment]
> @KZDKM you do know is quite broken when stuff like firefox is fullscreen, right? window is restored as fullscreen but the internal fullscreen state isn't triggered (in firefox's case, all bars remain visible, even when say a video was playing)

Yeah I know this is somehow broken when hyprland updated their internal fullscreen state logic, I will look into it.

**Comment by alba4k** [comment]
any updates on this? really annoying since I often slightly trigger the gesture while trying to change workspace (might change my config, sure, but that is a workaround not a fix)

**Comment by KZDKM** [comment]
pretty busy lately, will work on it soon
